src/adapter_wrap.cc

```diff
@@ -93,6 +122,16 @@ Handle<Value> AdapterWrap::OnNewListener(const Arguments& args) {
     obj->cec_callbacks.CBCecLogMessage = &OnCecLogMessage;
     obj->cec_adapter->EnableCallbacks((void *) obj->async_handle, &(obj->cec_callbacks));
   }
+  if (args[0]->Equals(String::New("command")) && obj->cec_callbacks.CBCecCommand == NULL) {
+        UvAsyncData* asyncData = new UvAsyncData;
+        asyncData->argThis = Persistent<Object>::New(args.This());
+        uv_mutex_init(&asyncData->mutex);
+        obj->async_handle = new uv_async_t;
+        obj->async_handle->data = (void *) asyncData;
+        uv_async_init(uv_default_loop(), obj->async_handle, OnUvAsync);
```

Comment:
wouldn't this overwrite the `async_handle`, what if there were two event listeners for both logmessage and command? I think it'd need a guard so it only calls this if there isn't already an async handle, or for us to store a new async handle for each message queue (although that seems very wasteful).